samples/config_helper_dawn.cc

```diff
@@ -43,12 +42,19 @@ amber::Result ConfigHelperDawn::CreateConfig(
   auto r = dawn::CreateMetalDevice(&dawn_instance_, &dawn_device_);
   if (!r.IsSuccess())
     return r;
-#else
-  return amber::Result("Can't make Dawn engine config");
+#else  // assuming VULKAN
+  dawn_instance_.DiscoverDefaultAdapters();
```

Comment:
Did we determine if the metal code above could move into this for loop?

Reply:
It doesn't work right now and we decided to leave it as it is.